cinema/views.py

```diff
@@ -57,7 +63,7 @@
 ):
     queryset = CinemaHall.objects.all()
     serializer_class = CinemaHallSerializer
-    authentication_classes = (TokenAuthentication,)
+
```

Comment:
The permission class `IsAdminOrIfAuthenticatedReadOnly` is correctly applied to the `CinemaHallViewSet`, ensuring that only admins can create cinema halls while authenticated users can read them.